http/src/main/resources/init-resources/gce-init.sh

```diff
@@ -376,7 +400,9 @@ if [ ! -z "$JUPYTER_DOCKER_IMAGE" ] ; then
   mkdir -p ${WORK_DIRECTORY}/packages
   chmod a+rwx ${WORK_DIRECTORY}/packages
 
-  # TODO: update this if we upgrade python version
+  # Install everything after having mounted the empty PD
+  # This should not be needed anymore if the jupyter home is a directory of the PD mount point
```

Comment:
+1 strong candidate for deletion, I can't imagine we still support runtimes where the home directory is not `/home/jupyter`

Reply:
What I mean here is that if we do not mount the PD in the home.directory then we can install these in the container directly